### PR TITLE
Point Verdant last minute bugfixes (chapter 1)

### PR DESCRIPTION
**Pull request by kyres1**
- bugfix: "Fixed invisible shelving on Point Verdant."
  - bugfix: "Fixed nonfunctioning animations on the Haneunim title screen."
  - tweak: "Disabled Point Verdant's rain due to missing functionality."

### CODE REVIEW

**Comment by kyres1** [comment]
!bugfix

File: maps/away/away_site/konyang/point_verdant/point_verdant_areas.dm [hunk 1]
```diff
@@ -120,22 +120,23 @@
...
 
-/area/point_verdant/outdoors/Initialize()
-	. = ..()
-	add_overlay(image("icon"='icons/effects/rain_effects.dmi',"icon_state"="splat","layer"=OBJ_LAYER-0.1))
-	add_overlay(image("icon"='icons/effects/rain_effects.dmi',"icon_state"="rain","layer"=MOB_LAYER+0.1))
+//Rainy areas are still incomplete, so these are commented out temporarily until a proper implementation is made
+///area/point_verdant/outdoors/Initialize()
+//	. = ..()
+//	add_overlay(image("icon"='icons/effects/rain_effects.dmi',"icon_state"="splat","layer"=OBJ_LAYER-0.1))
+//	add_overlay(image("icon"='icons/effects/rain_effects.dmi',"icon_state"="rain","layer"=MOB_LAYER+0.1))
```
Comment (NonQueueingMatt) [inline maps/away/away_site/konyang/point_verdant/point_verdant_areas.dm:130]: remove, don't comment

File: maps/away/away_site/konyang/point_verdant/point_verdant_areas.dm [hunk 2]
```diff
@@ -120,22 +120,23 @@
...
 	icon_state = "fitness_pool"
 
-/area/point_verdant/water/Initialize()
-	. = ..()
-	add_overlay(image("icon"='icons/effects/rain_effects.dmi',"icon_state"="ripple","layer"=OBJ_LAYER-0.1))
-	add_overlay(image("icon"='icons/effects/rain_effects.dmi',"icon_state"="rain","layer"=MOB_LAYER+0.1))
+///area/point_verdant/water/Initialize()
+//	. = ..()
+//	add_overlay(image("icon"='icons/effects/rain_effects.dmi',"icon_state"="ripple","layer"=OBJ_LAYER-0.1))
+//	add_overlay(image("icon"='icons/effects/rain_effects.dmi',"icon_state"="rain","layer"=MOB_LAYER+0.1))
```
Comment (NonQueueingMatt) [inline maps/away/away_site/konyang/point_verdant/point_verdant_areas.dm:139]: remove, don't comment

**Review by NonQueueingMatt — CHANGES_REQUESTED** [review]

File: code/modules/tables/rack.dm [hunk 3]
```diff
@@ -37,7 +37,7 @@
 /obj/structure/table/rack/retail_shelf
 	name = "retail shelves"
 	desc = "A large, assorted shelf with many platforms for setting things upon."
-	icon_state = "shelf"
+	icon_state = "shelf_alt"
```
Comment (Greenjoe12345) [inline code/modules/tables/rack.dm:40]: Shouldn't this be on a subtype of the shelves, so both types of shelf icon are usable?

**Review by NonQueueingMatt — APPROVED** [review]

**Comment by NonQueueingMatt** [comment]
!review

**Review by Arrow768 — APPROVED** [review]

**Pull request merged**